widget/table.go

```diff
@@ -15,50 +15,41 @@ const tableDividerThickness = 1
 // Declare conformity with Widget interface.
 var _ fyne.Widget = (*Table)(nil)
 
+// CellID is a type that represents a cell's position in a table based on it's row and column location.
+type CellID struct {
```

Comment:
nit: Missing collection prefix
```suggestion
// TableCellID is a type that represents a cell's position in a table based on it's row and column location.
type TableCellID struct {
```

Reply:
Resolved